### PR TITLE
Introduce ecs placementStrategy option

### PR DESCRIPTION
**Pull request by jasonletbetter**
The net effect here is to add support for the placementStrategy option with run_task.  It is backward compatible in that not specifying the option has no effect.

### CODE REVIEW

File: src/erlcloud_ecs.erl [hunk 1]
```diff
@@ -163,7 +163,10 @@
     volume_name_opt/0,
     volumes/0,
     volumes_opt/0,
-    volumes_opts/0
+    volumes_opts/0,
+    placement_strategy/0,
```
Comment (nalundgaard) [inline src/erlcloud_ecs.erl]: It looks like these are exported alphabetically. Any reason not to put them in accordingly?

File: include/erlcloud_ecs.hrl [hunk 2]
```diff
@@ -188,6 +193,7 @@
     memory :: undefined | pos_integer(),
     memory_reservation :: undefined | pos_integer(),
     mount_points :: undefined | [#ecs_mount_point{}],
+    placement_strategy :: undefined | [#ecs_placement_strategy{}],
```
Comment (nalundgaard) [inline include/erlcloud_ecs.hrl]: should this go after `name`?

File: src/erlcloud_ecs.erl [hunk 3]
```diff
@@ -492,7 +516,7 @@ container_volumes_from_opt() ->
                                     container_log_configuration_opt() |
                                     {memory, pos_integer()} |
                                     {memory_reservation, pos_integer()} |
-                                    container_mount_points_opt() |
+                                    container_mount_points_opt() |
```
Comment (nalundgaard) [inline src/erlcloud_ecs.erl]: 👾

File: include/erlcloud_ecs.hrl (hunk 2 above)
Comment (nalundgaard) [inline include/erlcloud_ecs.hrl]: where is the code to actually set this to something other than default? I don't see it in this PR.

Comment (jasonletbetter) [reply include/erlcloud_ecs.hrl]: It's a [run_task option](https://github.com/erlcloud/erlcloud/pull/547/files#diff-67e666f62a38974173c59f4bcd66bb1dR2175)

File: src/erlcloud_ecs.erl (hunk 1 above)
Comment (jasonletbetter) [reply src/erlcloud_ecs.erl]: sure

File: include/erlcloud_ecs.hrl (hunk 2 above)
Comment (jasonletbetter) [reply include/erlcloud_ecs.hrl]: actually, this is left over from a road not taken; deleting instead

**Pull request merged**